server/app-engine/static/vehicle-assignments.js

```diff
@@ -340,22 +342,22 @@ function handleKey(id) {
         util.log('- fromDate: ' + fromDate);
         util.log('- toDate  : ' + toDate);
 
-        for (var item of items) {
+        for (item of items) {
```

Comment:
Why is there no `let` here?

Reply:
updated - thanks for sharing the link about this